BUILDING.md

```diff
@@ -576,9 +576,9 @@ to run it again before invoking `make -j4`.
 
 * [Python 3.11](https://apps.microsoft.com/store/detail/python-311/9NRWMJP3717K)
 * The "Desktop development with C++" workload from
-  [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products) or
+  [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or
```

Comment:
Are we using vs2022 in our CI? I thought the only reason we're still recommending 2019 was the fact that 2022 wasn't fully supported. cc @nodejs/build

Reply:
In which sense would vs2022 not be fully supported? In the sense that we don't build with vs2022?

Reply:
Work is ongoing to update Windows VMs and compilers in the CI. The hope is to move to requiring VS2022 for Node.js 21 to support moving to C++20 (https://github.com/nodejs/node/pull/45427) but AFAIK the CI is not quite there yet.

See 
- https://github.com/nodejs/build/issues/3373
- https://github.com/nodejs/build/pull/3484
- https://github.com/nodejs/node/pull/49051

Reply:
@richardlau do you feel that it is OK for us to then get this PR merged as it is, even if CI is still using VS2019?

I've tested on an ARM64 VM and an x86 machine with W11 and both were able to build Node fine.